src/routes/__root.tsx

```diff
@@ -1,7 +1,7 @@
 import { createRootRoute, Outlet } from '@tanstack/react-router';
 import { TanStackRouterDevtools } from '@tanstack/router-devtools';
 
-import { Header } from '@/modules/Header';
+import { Header } from '@/modules';
```

Comment:
```suggestion
import { Header } from '@/modules/Header';
```

avoid using barrel files cause you might have a situation when all modules will be appended to one chunk

Reply:
Done